digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ProjectService.scala

```diff
@@ -1708,47 +1708,118 @@ class ProjectService(roadAddressService: RoadAddressService, roadLinkService: Ro
             (change, projectLinksInChange)
         }
 
-        if (projectLinks.isEmpty){
-          logger.error(s" There are no addresses to update, rollbacking update ${project.id}")
-          throw new InvalidAddressDataException(s"There are no addresses to update , rollbacking update ${project.id}")
+    if (projectLinks.isEmpty) {
+      logger.error(s" There are no addresses to update, rollbacking update ${project.id}")
+      throw new InvalidAddressDataException(s"There are no addresses to update , rollbacking update ${project.id}")
+    }
+
+    projectLinkDAO.moveProjectLinksToHistory(projectID)
+    logger.info(s"Moving project links to project link history.")
+    //        handleNewRoadNames(projectLinks, project)
```

Comment:
should commented code be removed?

Reply:
Yes. I removed the commented function call to 'handleNewRoadNames' since it is now called inside the try catch block.